src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = days * 40;
+
+  if (days <= 2) {
+    return rent;
+  }
+
+  if (days < 7) {
+    return rent - 20;
+  }
+
+  return rent - 50;
```

Comment:
not fixed comment about magic numbers.
write all numbers into constants and use them
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)
![Screenshot from 2023-05-26 12-30-16](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/33029864/7e3be3d0-1a73-472d-ab44-83c075cc708f)
![Screenshot from 2023-05-26 12-30-03](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/33029864/d5d5bb04-9f90-472d-976b-b2a13148be95)